dask/dataframe/tests/test_dataframe.py

```diff
@@ -2424,12 +2186,7 @@ def test_sorted_index_single_partition():
 def test_info():
```

Comment:
Maybe as a follow up, but could we modify `ddf.info()` to include the known / unknown information?

instead of

```python
In [47]: df.info()
<class 'dask.dataframe.core.DataFrame'>
Data columns (total 14 columns):
fl_date             object
unique_carrier      category
airline_id          int64
tail_num            object
...
```

something like

```python
In [47]: df.info()
<class 'dask.dataframe.core.DataFrame'>
Data columns (total 14 columns):
fl_date             object
unique_carrier      category[unknown]
airline_id          int64
tail_num            object
fl_num              int64
...
```